report.py

```diff
@@ -136,6 +140,20 @@ def run_report(config, blockers, server, header, test_email, no_email):
 	summary['total_unstable'] = "Total UNSTABLE: {}/{} = {}%".format(num_unstable, num_jobs, percent(num_unstable, num_jobs))
 	summary['total_failure'] = "Total FAILURE:  {}/{} = {}%".format(num_failure, num_jobs, percent(num_failure, num_jobs))
 
+	# create chart config
+	chart_config = {
+		'type': 'pie',
+		'data': {
+			'labels': ['Success', 'Unstable', 'Failed'],
+			'datasets': [{
+				'backgroundColor': ['green', 'yellow', 'red'],
+				'data': [int(num_success), int(num_unstable), int(num_failure)]
+			}]
+		}
+	}
+	encoded_config = quote(json.dumps(chart_config))
+	summary['chart_url'] = f'https://quickchart.io/chart?c={encoded_config}'
```

Comment:
Nothing for error state builds? [NO_KNOWN_BUILDS, ABORTED, ERROR]

Reply:
Do we get those in Jeeves report? The summary consists of only total failure/success/unstable I'd need examples so that i can put the statuses accordingly

Reply:
I will link some examples once I have restored the legacy reports to sealusa38

Reply:
What's the difference between Error and Failed jobs?

Reply:
FAILED are jobs that existed successfully in Jenkins but failed OpenStack deployment, ERROR are jobs that Jeeves couldn't get information from the Jenkins API for an unhandled reason